engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41700to41800.java

```diff
@@ -0,0 +1,87 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why is this here? I guess Nicolas already added upgrade path in main?

Reply:
I guess I made these changes when the upgrade path wasn't created yet, I'll update them, thanks!